website/docs/components/modal/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,352 @@
+## Usage
```

Comment:
Not a blocker per se and I recognize this is a complicated component, but there is _a lot_ of content here. Do you think it's possible to curate this more to lean into our desire for more brevity?

Reply:
@Dhaulagiri yeah that's fair. I'll do a bit of a content scrub to see what can be trimmed out.